README.md

```diff
@@ -1,2 +1,11 @@
-# neo3-boa-template
-A neo3-boa project template
+# Neo3-boa Template
+
+A [Neo3-boa](https://github.com/CityOfZion/neo3-boa) project template.
+
+Here, you'll find folders that contains dapp templates, inside each folder there will be a: 
```

Comment:
As we talked on Discord, we are having only 1 template on this repo, so let's move the files to the root and remove that folder.

I also want to have a "file tree" here on the readme instead of the bullet format.
Mention each folder and file names with a comment on the side, follow this example:

```
├── assets - This folder contains all the assets used on the compositions.
│   ├── images
│   │   └── Image1.png
│   └── videos
│       └── Video1.mp4
├── components - This folder contains the builder for each composition.
│   ├── Comp1.ts - The file name must match the composition name.
│   └── Comp2.ts
├── compositions - This folder contains the compositions used to create the videos.
│   ├── Comp1.aes
│   └── Comp2.aes
├── package.json - NPM package file.
└── tsconfig.json - Typescript configuration file.

```